.ci/run.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+import stat
+import platform
+import subprocess
+import tempfile
+import logging
+from contextlib import contextmanager
+from tabulate import tabulate
+import colorama
+
+
+FAIL_FAST = os.getenv("FAIL_FAST", "0").lower() in ["1", "y", "yes", "true"]
+LOGGING_LEVEL = int(os.getenv("CONAN_LOGGING_LEVEL", logging.INFO))
+logging.basicConfig(format='%(asctime)s [%(levelname)s]: %(message)s', level=LOGGING_LEVEL)
+
+
+@contextmanager
+def chdir(dir_path):
+    current = os.getcwd()
+    os.chdir(dir_path)
+    logging.debug("cd {}".format(dir_path))
+    try:
+        yield
+    finally:
+        logging.debug("cd {}".format(current))
+        os.chdir(current)
+
+
+def writeln_console(message):
+    sys.stdout.write(message)
+    sys.stdout.write('\n')
+    sys.stderr.flush()
+    sys.stdout.flush()
+
+
+def get_build_list():
+    builds = []
+    script = "build.bat" if platform.system() == "Windows" else "build.sh"
+    for root, dirs, files in os.walk("."):
+        if "build.py" in [os.path.basename(file) for file in files]:
+            builds.append(os.path.join(root, "build.py"))
+            continue
+
+        for file in files:
+            if os.path.basename(file) == script:
+                builds.append(os.path.join(root, file))
+
+    return builds
+
+
+def chmod_x(script):
+    logging.debug("chmod +x {}".format(script))
+    st = os.stat(script)
+    os.chmod(script, st.st_mode | stat.S_IEXEC)
+
+
+def get_conan_env():
+    temp_folder = tempfile.mkdtemp(prefix="conan-", suffix="-home")
+    os.environ["CONAN_USER_HOME"] = temp_folder
+    logging.debug("CONAN_USER_HOME: {}".format(temp_folder))
+    return os.environ
+
+
+def configure_profile():
+    subprocess.check_output("conan profile new default --detect",
+                            stderr=subprocess.STDOUT,
+                            shell=True)
+    if platform.system() == "Linux":
+        subprocess.check_output("conan profile update settings.compiler.libcxx=libstdc++11 default",
+                                stderr=subprocess.STDOUT,
+                                shell=True)
+
+
+def print_build(script):
+    dir_name = os.path.dirname(script)
+    dir_name = dir_name[2:] if dir_name.startswith('.') else dir_name
+    writeln_console("================================================================")
+    writeln_console("* " + colorama.Style.BRIGHT + "{}".format(dir_name.upper()))
+    writeln_console("================================================================")
+
+
+def run_scripts(scripts):
+    results = {}
```

Comment:
We may initialize the dictionary so we get the table with all the examples at the bottom, and keep the order of the scripts in the table:

```suggestion
    results = OrderedDict.fromkeys(scripts, '')
```

Reply:
done